mdgo/mdgopackmol.py

```diff
@@ -77,7 +88,7 @@ def run_packmol(self):
         """Run packmol and write out the packed structure."""
         try:
             p = subprocess.run(
-                "packmol < {}".format(self.input),
+                "packmol < '{}'".format(self.input),
```

Comment:
I don't think this will work. I did a bunch of manual testing with packmol, and the problem is that when reading the input file, packmol does not respect normal terminal conventions for handling spaces like quoting the path or escaping the spaces.

So, to be clear, the change I made here does allow you to save the output file in a path with spaces, but you can't reference structure files in paths with spaces

Reply:
I see, thanks @rkingsbury ! Will merge the current version and patch fixes in the future!